perfmetrics/scripts/continuous_test/gcp_ubuntu/e2e_tests/build.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+# Copyright 2023 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This will stop execution when any command will have non-zero status.
+
+set -e
+sudo apt-get update
+
+# e.g. architecture=arm64 or amd64
+architecture=$(dpkg --print-architecture)
+echo "Installing git"
+sudo apt-get install git
+echo "Installing go-lang 1.20.5"
+wget -O go_tar.tar.gz https://go.dev/dl/go1.20.5.linux-${architecture}.tar.gz -q
+sudo rm -rf /usr/local/go && tar -xzf go_tar.tar.gz && sudo mv go /usr/local
+export PATH=$PATH:/usr/local/go/bin
+echo "Installing docker "
+sudo mkdir -p /etc/apt/keyrings
```

Comment:
Can we keep a common script for running integration tests for all CI tests (periodic and presubmits) ? The script to also install required dependencies for integration tests and can take branch name from which to build gcsfuse ?

That way the same bucket problem mentioned in the other commit will also be resolved.

Reply:
1. I thought that but creating script will only contain one line to run integration test, as all the other dependency are require for fio test also, so we will keep there only.

Reply:
- I believe we should try not to replicate same logic at multiple places. Given that you can do:
    - Move the entire logic of integration test including dependency installation, random bucket creation and tests themselves to another script which can be used for both periodic and perf test.
    - The script can take branch and a bool parameter whether to run integration test by first installing package using docker and using --testInstalledPackage or simply leave it to go test to take care of.
    -  Move the dependency installation of fio load to https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/perfmetrics/scripts/presubmit/run_load_test_on_presubmit.sh. So, your build.sh will only contain logic to either call perf test script or integration test script based on tag.
- That will solve bucket problem because the common script (the existing presubmit integration test script) already contains logic for creating random bucket.

Reply:
got it. I don't think so branch parameter is require as it will checkout to the branch from parent script only.